prebuild/downloadMissions.ts

```diff
@@ -0,0 +1,22 @@
+import { dirname } from 'node:path';
```

Comment:
can we do that for all locales instead of de_DE only? locales should be available from `src/config.ts` iirc.

Reply:
Missions will be now downloaded for every language.